utils/dwarf.c

```diff
@@ -310,6 +311,7 @@ static int setup_dwarf_info(const char *filename, struct uftrace_dbg_info *dinfo
 		pr_dbg2("failed to setup debug info: %s\n", dwfl_errmsg(dwfl_errno()));
 		dwfl_end(dinfo->dwfl);
 		dinfo->dwfl = NULL;
+		close(fd); // Close the file descriptor to release the acquired resource
```

Comment:
I thought `dwfl_end()` would close the fd since it can remember the fd from the c`dwfl_report_offline()`.  Can you please check if it's the case?

Reply:
You are right, it will be handled by dwfl_end()